style.css

```diff
@@ -0,0 +1,161 @@
+/* *{
```

Comment:
This could be removed to keep up with HTML and CSS best practices! Otherwise, great job